src/main/java/uk/gov/ons/json/TokeniserResponse.java

```diff
@@ -23,4 +23,5 @@
 	private String postcode;
 	private String postcodeIn;
 	private String postcodeOut;
+	private String lpiLogicalStatus;
```

Comment:
You don't want this here. This represents the parser's response which doesn't include the lpilogicalstatus.